### DISCUSSION TITLE
Worried about Flutter

### DISCUSSION THREAD

**Discussion by malyzkhan** [community/community#64270] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Is Flutter is good enough?

**Comment by inaamulhaqdev** [comment, marked as answer]
Yees. Flutter is good enough.

**Comment by Binary-Ninja-007** [comment]
Flutter is very Good!!
One of the benefits of using Flutter is that you have access to a wide variety of pre-built widgets that you can use in your app. This saves you valuable time and resources that would otherwise be spent on developing custom widgets from scratch.
If you want to know more, I will explain for you.

**Reply by malyzkhan** [reply]
Thank you jack
